src/main/java/com/bamdoliro/maru/application/fair/QueryFairDetailUseCase.java

```diff
@@ -1,20 +1,36 @@
 package com.bamdoliro.maru.application.fair;
 
+import com.bamdoliro.maru.domain.fair.domain.Attendee;
 import com.bamdoliro.maru.domain.fair.domain.Fair;
 import com.bamdoliro.maru.infrastructure.persistence.fair.AttendeeRepository;
 import com.bamdoliro.maru.presentation.fair.dto.response.FairDetailResponse;
 import com.bamdoliro.maru.shared.annotation.UseCase;
 import lombok.RequiredArgsConstructor;
 
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
 @RequiredArgsConstructor
 @UseCase
 public class QueryFairDetailUseCase {
 
     private final FairFacade fairFacade;
     private final AttendeeRepository attendeeRepository;
 
-    public FairDetailResponse execute(Long fairId) {
+    public FairDetailResponse execute(Long fairId, String sort) {
         Fair fair = fairFacade.getFairDetail(fairId);
-        return new FairDetailResponse(fair, attendeeRepository);
+
+        List<Attendee> sortedAttendeeList = switch (sort) {
+            case "name_asc" -> fair.getAttendeeList().stream()
+                .sorted(Comparator.comparing(Attendee::getName))
+                .collect(Collectors.toList());
```

Comment:
P5) 진짜진짜 사소한건데 현재 조회용 리스트를 반환하고 있으니 가변 리스트를 반환해주는 Collectors.toList() 보다는 불변리스트를 반환해주는 .toList() 해주는 것도 좋을 거 같아요.
```suggestion
                .toList();
```

Reply:
반영하겠습니당